openapi.json

```diff
@@ -1,1096 +1,1322 @@
 {
-  "openapi": "3.0.3",
-  "info": {
-    "title": "Littil backend API",
-    "contact": {
-      "name": "Littil",
-      "url": "https://littil.org/contact",
-      "email": "info@littil.org"
+  "openapi" : "3.0.3",
```

Comment:
Which formatter do you use for json? Need to add it to the project, so we don't get these kinds of differences in json files whenever we touch something in the file